.github/workflows/kfp-sdk-tests.yml

```diff
@@ -0,0 +1,46 @@
+name: KFP SDK Tests
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'sdk/**'
+      - 'test/presubmit-tests-sdk.sh'
```

Comment:
I think this can be removed.

Reply:
Done, removed this.

Reply:
I still see it.

Reply:
Pushed again, changes should be visible now.